Userland/Libraries/LibCore/IOSurface.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/OwnPtr.h>
+#include <LibCore/IOSurface.h>
+
+#if !defined(AK_OS_MACOS)
+static_assert(false, "This file must only be used for macOS");
+#endif
+
+#define FixedPoint FixedPointMacOS
+#define Duration DurationMacOS
+#include <IOSurface/IOSurface.h>
+#undef FixedPoint
+#undef Duration
```

Comment:
bleh. still hate this hack hah. We should create an issue to resolve it, I don't remember if there was one in the serenity repo already.